server/api/urls.py

```diff
@@ -45,4 +45,7 @@
 
     url(r'^event/(?P<pk>[0-9]+)/?$', views.EventDetail.as_view(),
         name='event_detail'),
+
+    url(r'^oauthcallback/?$', views.OauthCallback.as_view(),
```

Comment:
@OsayamenOsaretin Please In what context or where will this be used?

Reply:
This is a callback for the Oauth2 authorization flow that is called by the authorization server from Google when the user grants or doesn't grant access to our app.

Reply:
@OsayamenOsaretin Okay.